README.md

```diff
@@ -66,6 +65,8 @@ GPflow leaves installing tensorflow to the user so that you can choose whether y
 
 ## Getting Started with GPflow 2.0
 
+*Note that GPflow 2.0 has not yet had an official release and interfaces, function signatures, variable names etc. may still change. Likewise, we are open to suggestions for how to improve the design of gpflow2!*
```

Comment:
"GPflow 2.0" and "gpflow2" both used - use one format?

Reply:
We can unify this in another PR?